.github/workflows/main.yml

```diff
@@ -27,8 +29,6 @@ jobs:
             image_tag: "3.9"
           - python: "3.7"
             image_tag: "3.8"
-          - python: "3.8"
-            image_tag: "3.7"
```

Comment:
I think, we decided to keep compatibility with 5 versions of pulpcore.
OTOH 3.7 seems to get bugfix update still. Should we keep it around?

Reply:
Good question. I believe Katello and galaxy are both on 3.7 so I suppose we should.